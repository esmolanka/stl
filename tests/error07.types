
type Foo = Unit

type Foo = <Other>

