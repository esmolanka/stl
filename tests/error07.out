tests/error07.stl:4:6-4:9: error:
    Duplicate type definition 'Foo'. It has been already defined at:
    tests/error07.stl:2:12-2:16